### PR TITLE
Animals2

### PR DESCRIPTION
**Pull request by TheShadowOfHassen**
More animals!

### CODE REVIEW

File: 0-ad-history-mod-package/simulation/templates/gaia/fauna_wolf.xml [hunk 1]
```diff
@@ -24,6 +24,10 @@
     <GenericName>Wolf</GenericName>
     <SpecificName>Canis lupus</SpecificName>
     <Icon>gaia/fauna_wolf.png</Icon>
+    <History>
+        While their size and color varied from location, wolves in the ancient world are one species, Canis Lupis. Wolves are canine, predators who run in packs, consisting of extended family. Together they live,  hunt and raise the next generation of wolves.
+In recent time some people have begun to question if wolves wolves are as aggressive as they are sometimes portrayed, however the coordination in packs allows them to take down animals far bigger than themselves, and if the fancy struck them could easy kill a gatherer who was working by himself.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/gaia/fauna_wolf.xml:29]:
```suggestion
       In recent times, some people have begun to question if wolves wolves are as aggressive as they are sometimes portrayed, however, the coordination in packs allows them to take down animals far bigger than themselves, and if the fancy struck them, they could easily kill a gatherer who was working by himself.
```

File: 0-ad-history-mod-package/simulation/templates/gaia/fauna_wolf.xml [hunk 2]
```diff
@@ -24,6 +24,10 @@
     <GenericName>Wolf</GenericName>
     <SpecificName>Canis lupus</SpecificName>
     <Icon>gaia/fauna_wolf.png</Icon>
+    <History>
+        While their size and color varied from location, wolves in the ancient world are one species, Canis Lupis. Wolves are canine, predators who run in packs, consisting of extended family. Together they live,  hunt and raise the next generation of wolves.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/gaia/fauna_wolf.xml:28]:
```suggestion
        While their size and color varied from location, wolves in the ancient world are one species, Canis Lupis. Wolves are canine predators who run in packs consisting of extended family. Together, they live, hunt and raise the next generation of wolves.
```

**Review by indoptogopt — CHANGES_REQUESTED** [review]

**Comment by TheShadowOfHassen** [comment]
Is this good?

**Comment by indoptogopt** [comment]
Yes

**Pull request merged**